### PR TITLE
updated references and fixed typo

### PR DESCRIPTION
**Pull request by josephdviviano**
reference fixes & typos from previous PR

### CODE REVIEW

File: src/gfn/losses/trajectory_balance.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ class TBParametrization(PFBasedParametrization):
 class TrajectoryBalance(TrajectoryDecomposableLoss):
     """Loss object to evaluate the TB loss on a batch of trajectories.
 
-    This method is described in x.xx of [REFERENCE HERE](https://arxiv.org/abs/???))
+    This method is described in 2.3 of
```
Comment (saleml) [inline src/gfn/losses/trajectory_balance.py:33]: this is subtb :D

File: src/gfn/losses/sub_trajectory_balance.py [hunk 2]
```diff
@@ -19,8 +19,8 @@ class SubTBParametrization(PFBasedParametrization):
 class SubTrajectoryBalance(TrajectoryDecomposableLoss):
     """Loss object to evaluate the Sub Trajectory Balance Loss.
 
-    This method is described in section x.xx of
-    [THIS PAPER](https://arxiv.org/abs/???))
+    This method is described in section 3 of
```
Comment (saleml) [inline src/gfn/losses/sub_trajectory_balance.py:22]: this is TB

File: src/gfn/losses/trajectory_balance.py (hunk 1 above)
Comment (josephdviviano) [reply src/gfn/losses/trajectory_balance.py:33]: omg 🤦 sorry

**Comment by josephdviviano** [comment]
thanks -- should be fixed now (subtb and tb in their right place)

**Pull request merged**